src/po/Make_mvc.mak

```diff
@@ -28,9 +24,9 @@ VIM = ..\vim
 # Correct the following line for the directory where gettext et al is installed
 GETTEXT_PATH = D:\Programs\GetText\bin
 
-MSGFMT = $(GETTEXT_PATH)\msgfmt -v
-XGETTEXT = $(GETTEXT_PATH)\xgettext
-MSGMERGE = $(GETTEXT_PATH)\msgmerge
+MSGFMT = "$(GETTEXT_PATH)\msgfmt" -v
+XGETTEXT = "$(GETTEXT_PATH)\xgettext"
+MSGMERGE = "$(GETTEXT_PATH)\msgmerge"
 
```

Comment:
That's weird. I checked now, no changes have been made to the "gettext" path definition.
Maybe the formatting of the lines in the command call was affected, but it shouldn't have been. That's weird. I'll check.

Reply:
As you can see, the first error is thrown at the "iconv.exe" file availability check line. And you use escape spaces in the "gettext" path when calling nmake.
```
E:\Project\vim\en-ru\po\tst\tst2>E:\Project\vim\tools\nmake.exe -f Make_mvc.mak GETTEXT_PATH="""D:\Program Files\GetText\bin"""

Microsoft (R) Program Maintenance Utility Version 14.27.29111.0
Copyright (C) Microsoft Corporation.  All rights reserved.

Make_mvc.mak(39) : fatal error U1023: syntax error in expression
Stop.
```
Well, this is a well-known and old DOS and Windows error from the time when FAT32 support was introduced (and maybe even earlier).

Because of my old habit and following the documentation recommendations, I enclose paths with spaces and other special characters in quotes.

> Long Filenames
> You can use long filenames if they are supported by your file system. However,
> you must enclose the name in double quotation marks, as in the following
> dependency line:
> all : "VeryLongFileName.exe"

> EXIST (path)
> Evaluates to true if path exists. EXIST can be used in complex expressions
> using binary logical operators. If path contains spaces (allowed in some file
> systems), enclose it in double quotation marks.

And you use quotation marks. That is, we get double escaping. That's why the error occurs.
Yes, I should have foreseen this and commented on it and changed the code in the makefile.
Well, it should work fine now. Thank you for fixing it!    

       
 
  By the way, instead of using GETTEXT_PATH="\\"C:\Program Files\Git\usr\bin\\"", you can use GETTEXT_PATH="""C:\Program Files\Git\usr\bin\""". It probably looks weird too, but it works the same way.

Reply:
@k-takata , Please enclose the $(GETTEXT_PATH) macro in quotes to avoid errors when updating PO files.
```
# Don't add a dependency here, we only want to update the .po files manually
$(LANGUAGES):
	@$(MAKE) -nologo -f Make_mvc.mak $(PACKAGE).pot GETTEXT_PATH="$(GETTEXT_PATH)"
	$(CP) $@.po $@.po.orig
	$(MV) $@.po $@.po.old
	$(MSGMERGE) $@.po.old $(PACKAGE).pot -o $@.po
	$(RM) $@.po.old
```